pom.xml

```diff
@@ -16,13 +16,13 @@
 	<groupId>org.eclipse.reddeer</groupId>
 	<artifactId>parent</artifactId>
 	<name>RedDeer Parent POM File</name>
-	<version>4.5.0-SNAPSHOT</version>
+	<version>4.6.0-SNAPSHOT</version>
 	<packaging>pom</packaging>
 
 	<properties>
 		<tycho-version>2.7.5</tycho-version>
 		<tychoExtrasVersion>${tycho-version}</tychoExtrasVersion>
-		<eclipse-target-site>https://download.eclipse.org/releases/2023-03/</eclipse-target-site>
+		<eclipse-target-site>https://download.eclipse.org/releases/2023-06/</eclipse-target-site>
 		<swtbot-update-site>https://download.eclipse.org/technology/swtbot/snapshots/</swtbot-update-site>
         <!-- when making final release, switch from S to R latest update site - https://github.com/eclipse/reddeer/issues/2100 -->
         <eclipse-orbit-update-site>https://download.eclipse.org/tools/orbit/downloads/latest-R/</eclipse-orbit-update-site>
```

Comment:
Please see commented eclipse orbit repo url. During the development you should be using `latest-S` and switch to `latest-R` before final milestone release.

Reply:
Done